lib/mv_opentelemetry/span_tracer.ex

```diff
@@ -0,0 +1,146 @@
+defmodule MvOpentelemetry.SpanTracer do
+  @moduledoc """
+  Reusable behaviour for listening to events emmited with `:telemetry.span/3` convention and
+  converting them into OpenTelemetry traces.
+
+  You can define custom tracers modules and then register them at the start of your application.
+  At minimum, you are required to implement `c:handle_event/4`, which has exactly the same
```

Comment:
And at maximum?

Reply:
Nah, you can also override the `c:register_tracer/1`. I think at least would be a better wording, will update.